index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ja">
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" href="/favicon.svg" />
```

Comment:
ここで拡張子の設定を直す必要があると思います
現時点だとファイルの参照先がなさそう
```suggestion
    <link rel="icon" href="/favicon.ico" />
```